onnxruntime/python/tools/transformers/gpt2_helper.py

```diff
@@ -407,7 +407,6 @@ def export_onnx(model,
                           f=onnx_model_path,
                           input_names=input_names,
                           output_names=output_names,
-                          example_outputs=outputs,
```

Comment:
My worry is whether the new code will continue to work well with pytorch 1.10 and pytorch 1.9. If the parameter is not necessary, initially we didn't need to have it. Which means there is some garbage code around it that needs be cleaned. Intuitively, I feel it isn't the case.  For what I know, right now we can't drop the support for pytorch 1.10 yet. 

So before this PR gets merged, we need to make it clear which pytorch versions onnxruntime should support.  @askhade  FYI.

Reply:
ORT training package has two primary python modules:
1. `ORTTrainer`
2. `ORTModule`

While we continue supporting `ORTModule` with the latest torch version, we probably want to come up with a plan to deprecate `ORTTrainer`. So, we should move torch version to 1.11 for ORTModule, and continue using 1.10 for ORTTrainer until we phase out ORTTrainer and archive it. Until then, we probably will have to support two versions of torch. We can have a more detailed discussion on this.

Reply:
@snnn yes this code will work with torch 1.10 and torch 1.9.0. We have all the removed example_ouput commits in this PR - https://github.com/microsoft/onnxruntime/pull/10864 and the CIs have passed.